bower.json

```diff
@@ -32,15 +32,14 @@
     "url": "git://github.com/patternfly/angular-patternfly.git"
   },
   "dependencies": {
-    "angular": "1.2.25 - 1.3.*",
-    "patternfly": "~2.0.0",
-    "c3": "~0.4.10"
```

Comment:
> It is but it comes in with base patternfly so I don't think we want it to get pulled twice.

It won't be downloaded twice, bower handles that.  What happens if patternfly ever drops the dependency of c3?  It's unlikely to happen, especially without warning, but I think it's a good practice to include as dependencies libraries that you are using directly.